### DISCUSSION TITLE
[hopefully fixed!] ca-load container fails to start

### DISCUSSION THREAD

**Discussion by yzhong32** [camicroscope/caMicroscope#715] [Q&A] [answered]
After cloning the Distro repository, I attempted to use the command docker-compose -f develop.yml up to build and run the containers. However, I faced a consistent issue where the ca-load container failed to start, exiting with an error message. Has anyone else faced the same issue and knows how to solve it?
<img width="827" alt="Screenshot 2024-03-09 at 10 48 13 PM" src="https://github.com/camicroscope/caMicroscope/assets/143434843/fdb4ce2c-e4fe-41c0-b966-60844a745db9">

<img width="837" alt="Screenshot 2024-03-09 at 10 50 09 PM" src="https://github.com/camicroscope/caMicroscope/assets/143434843/3a941ff5-c228-4e18-9754-848b68d400dd">

**Comment by gbengaoluwadahunsi** [comment]
Hello @yzhong32 ,
same here like many others, we are trying to fix that, we might have to call the attention of @birm  to guide us through.

**Comment by birm** [comment]
Can someone test if a recent change that I've done fixes this for you by running the build command again? This has been a lot more trouble than expected!

**Reply by We13b-MD** [reply]
i will test it soon working on it

**Comment by gbengaoluwadahunsi** [comment]
Ok Birm

**Comment by gbengaoluwadahunsi** [comment]
Hello @birm,
I forked, cloned, and ran the Distro app again and still having the  same error, the ca-loader  is giving an error after uploadding the image.

**Reply by birm** [reply, marked as answer]
Can you please run `docker-compose -f develop.yml build loader --no-cache` then restart it in hopes it helps?

**Comment by yzhong32** [comment]
> Can you please run `docker-compose -f develop.yml build loader --no-cache` then restart it in hopes it helps?

Hello Birm , after executing the rebuild and restart commands, the ca-load container is now operational.

**Comment by gbengaoluwadahunsi** [comment]
Hello @birm, ca loader is working perfectly now
Thanks
